config/config_test.go

```diff
@@ -166,6 +168,65 @@ func TestDefaultConfig(t *testing.T) {
 	os.Unsetenv("DOCKER_DD_AGENT")
 }
 
+func setupSecretScript(script string) error {
```

Comment:
since we always use the same executable, consider removing the argument and wrapping this in sync.Once() so we only have to compile the program once